app/views/user_email_changed_mailer/changed_email.html.erb

```diff
@@ -0,0 +1,14 @@
+<p>
+  <%= @user.name %>,
+</p>
+
+<p>
+  Your email address has changed at
+  <%= ActionMailer::Base.default_url_options[:host] %> from
+  <%= @user.original_email %> to <%= @user.email %>.
```

Comment:
If I read your test correctly old emails are going to be able to return an array? and if not, how many old emails are we storing? I would think we would store them all, no? and if it is returning an array then we probably want to do ```@user.original_email.last``` right?